Testscripts/Windows/GPU-DRIVER-INSTALL.ps1

```diff
@@ -126,7 +126,7 @@ function Main {
             $currentTestResult.TestResult = Get-FinalResultHeader -resultarr "ABORTED"
             return $currentTestResult
         }
-        $CurrentTestResult.TestSummary += New-ResultSummary -metaData "Using nVidia driver: $driver" -testName $CurrentTestData.testName
+        $CurrentTestResult.TestSummary += New-ResultSummary -metaData "Using nVidia driver" -testName $CurrentTestData.testName -testResult $driver
```

Comment:
I need that metaData text as it displays the use of either CUDA or GRID driver in the test result summary.
Why is that conflicting with the DB insert?

Reply:
Metadat is "Using nVidia driver" and it's value is - "GRID/CUDA".
-testResult is mandetry in "New-ResultSummary" fuction Because, this function creates line  as - 
Metadata : Value.
If you don't give -Test result, the the line created is Metadata : (blank). This gives problem at parsing the result when uploading result to DB.
By the way, I've not removed the text. I just moved to correct place.

Execution Log:

```
   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes) 
-------------------------------------------------------------------------------------------------------------------------------------------
    1 GPU                  NVIDIA-GRID-DRIVER-VALIDATION-Standard_NV6                                        PASS                 12.3 
	Using nVidia driver : GRID 
	lsvmbus: Expected "PCI Express pass-through" count: 1, count inside the VM: 1 : PASS 
	lspci: Expected "3D controller: NVIDIA Corporation" count: 1, found inside the VM: 1 : PASS 
	lshw: Expected Display adapters: 1, total adapters found in VM: 1 : PASS 
	nvidia-smi: Expected GPU count: 1, found inside the VM: 1 : PASS 
    2 GPU                  NVIDIA-GRID-DRIVER-VALIDATION-Standard_NV12                                       PASS                11.48 
	Using nVidia driver : GRID 
	lsvmbus: Expected "PCI Express pass-through" count: 2, count inside the VM: 2 : PASS 
	lspci: Expected "3D controller: NVIDIA Corporation" count: 2, found inside the VM: 2 : PASS 
	lshw: Expected Display adapters: 2, total adapters found in VM: 2 : PASS 
	nvidia-smi: Expected GPU count: 2, found inside the VM: 2 : PASS 
    3 GPU                  NVIDIA-GRID-DRIVER-VALIDATION-Standard_NV24                                       PASS                13.99 
	Using nVidia driver : GRID 
	lsvmbus: Expected "PCI Express pass-through" count: 4, count inside the VM: 4 : PASS 
	lspci: Expected "3D controller: NVIDIA Corporation" count: 4, found inside the VM: 4 : PASS 
	lshw: Expected Display adapters: 4, total adapters found in VM: 4 : PASS 
	nvidia-smi: Expected GPU count: 4, found inside the VM: 4 : PASS 
```

Reply:
Thanks for the clarification Shital, this makes sense now.